src/EnergyPlus/WindowManager.cc

```diff
@@ -7025,11 +7269,13 @@ namespace WindowManager {
         Real64 TiltDeg;         // glazing tilt in degrees
         Real64 sineTilt;        // sine of glazing tilt
         Real64 Nuint;           // Nusselt number for interior surface convection
+        Real64 CoeffAdjRatio;   // Convective and radiative adjustment ratio for exterior window surfaces
```

Comment:
No need to declare local variables at the top of the function. That's a legacy coding style.

Reply:
Removed the declaration